### PR TITLE
updated A API docs

### PR DESCRIPTION
**Pull request by jaredatron**
Rendered here: [Updated A API Server Docks.zip](https://github.com/jlinclabs/apidocs/files/2562623/Updated.A.API.Server.Docks.zip)

### CODE REVIEW

File: a_server.md [hunk 1]
```diff
@@ -1,821 +1,707 @@
...
-mailingpostalcode
-mailingstate
-mailingcountry
-phone
-homephone
-mobilephone
-email
-birthdate
-gender
+POST /api/organization/:organizationApikey/feed/posts/delete
```
Comment (nodatall) [inline a_server.md]: `Post body` and `Response body` are missing for this endpoint

File: a_server.md [hunk 2]
```diff
@@ -1,821 +1,707 @@
...
+    "createdAt": "2018-11-08T00:54:27.389Z",
+    "organizationPost": false
+  }
+}
+```
 
-If `all_share` is set to false, no updates are shared with the B server, regardless of the settings of the other _share fields.
 
-{% include starsep.html %}
+### Create an Organization Feed Post
```
Comment (nodatall) [inline a_server.md]: Missing example `Post body`

File: a_server.md [hunk 3]
```diff
@@ -1,821 +1,707 @@
...
 {
- "success": true,
- "signuptoken": "existing"
+  "success": true
 }
 ```
 
-{% include starsep.html %}
+At this point an email will be sent to the given email address.
+The email will contain a link to that will verify the the given email.
```
Comment (nodatall) [inline a_server.md]: `link to that` -> `link that`

File: a_server.md [hunk 4]
```diff
@@ -1,821 +1,707 @@
...
 ```json
 {
- "token": "de7a126fef783bd6d9ac1da48cada93ee"
+  "resetPasswordToken": "GET_THIS_FROM_YOUR_EMAIL",
+  "password": "newPassword",
+  "passwordConfirmation": "newPassword"
 }
 ```
-On success it returns just:
+
```
Comment (nodatall) [inline a_server.md:165]: I'd add a sentence here that lets people know an email is sent.

File: a_server.md [hunk 5]
```diff
@@ -1,821 +1,707 @@
...
-The format is /api/vendors/{login_token}
-
-_GET https://sandbox.a.jlinclabs.net/api/vendors/de7a126fef783bd6..._
+### Get a Page of SISA Events for an Organization
 
-Which returns:
+Endpoint:
+```
+GET /api/organizations/sisa_events/:organizationApikey?page=1
+```
```
Comment (nodatall) [inline a_server.md:519]: Add params here:
```
page (number)
```

**Review by nodatall — COMMENTED** [review]
🍞

File: a_server.md (hunk 4 above)
Comment (jaredatron) [reply a_server.md:165]: an email is sent here? Isnt this the enpoint that actually changes the password?

**Comment by jaredatron** [comment]
Fixed all issues from @nodatall 
[Updated A API Server Docks.zip](https://github.com/jlinclabs/apidocs/files/2563329/Updated.A.API.Server.Docks.zip)

**Pull request merged**